pkgs/applications/misc/dunst/default.nix

```diff
@@ -38,6 +38,10 @@ stdenv.mkDerivation rec {
   postInstall = ''
     wrapProgram $out/bin/dunst \
       --set GDK_PIXBUF_MODULE_FILE "$GDK_PIXBUF_MODULE_FILE"
+
+    install -D contrib/_dunst.zshcomp $out/share/zsh/site-functions/_dunst
+    install -D contrib/_dunstctl.zshcomp $out/share/zsh/site-functions/_dunstctl
+    sed -i "s@jq -M@${jq}/bin/jq -M@" $out/share/zsh/site-functions/_dunstctl
```

Comment:
`substituteInPlace` is better here because it will give a warning if no replacement has been done. and this instances does not need regex.

Reply:
Done.